src/notifications/notificatielog/templates/base.html

```diff
@@ -0,0 +1,56 @@
+{% load static %}<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
```

Comment:
This means you cannot use it on a localhost/offline setup. Not a dealbreaker, but could be a problem for demo's/offline development

Reply:
Will resolve later.